<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plot_scatter_widget</class>
 <widget class="QWidget" name="plot_scatter_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="scatter_label_title">
       <property name="text">
        <string>Plot Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="scatter_line_title"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="scatter_label_x">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>x variable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="scatter_cb_x"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="scatter_label_y">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>y variable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="scatter_cb_y"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
